imports/plugins/core/graphql/server/no-meteor/resolvers/cart/Mutation/index.js

```diff
@@ -5,7 +5,6 @@ import removeCartItems from "./removeCartItems";
 import setEmailOnAnonymousCart from "./setEmailOnAnonymousCart";
 import setShippingAddressOnCart from "./setShippingAddressOnCart";
 import updateCartItemsQuantity from "./updateCartItemsQuantity";
-import updateFulfillmentOptionsForGroup from "./updateFulfillmentOptionsForGroup";
```

Comment:
Did this get moved or deleted?

Reply:
It was moved to being directly imported. In general I want to move away from calling all these mutations from context and rather import them directly where they are called, but some will still be put on context so that other plugins can call them. I'm changing this yet again and straightening out all the inconsistencies as part of getting back to the registerPlugin structure as we discussed.